tests/test-cases/include-inputs/integration.test.ts

```diff
@@ -244,6 +245,27 @@ scan-website:
     expect(writeStreams.stdoutLines[0]).toEqual(expected);
 });
 
+test("include-inputs for type boolean (falsy)", async () => {
+    const writeStreams = new WriteStreamsMock();
+    await handler({
+        cwd: "tests/test-cases/include-inputs/input-templates/types/boolean",
+        file: ".gitlab-ci-2.yml",
+        preview: true,
+        jsonSchemaValidation: true, // this test depends on the json schema validation, do not set to false
+    }, writeStreams);
+
+    const expected = `---
+stages:
+  - .pre
+  - test
+  - .post
+scan-website:
+  script:
+    - echo false`;
+
+    expect(writeStreams.stdoutLines[0]).toEqual(expected);
+});
```

Comment:
```suggestion
    const expected = [
        "---",
        "stages:",
        "  - .pre",
        "  - test",
        "  - .post",
        "scan-website:",
        "  script:",
        "    - echo false",
    ];

    expect(writeStreams.stdoutLines[0]).toEqual(expected.join("\n"));
});
```

Reply:
do you still want me to apply the suggestion ? i see that you've approve the pr

personally i prefer the backticks version because for ease of (typing/copy pasting)...

Reply:
I don't prefer the weird indentation or backticks, but I can live with both :smile: